includes/Admin/Rest/WC_REST_Square_Settings_Controller.php

```diff
@@ -150,6 +151,11 @@ public function register_routes() {
 						'type'              => 'string',
 						'sanitize_callback' => '',
 					),
+					'enable_order_fulfillment_sync'    => array(
+						'description'       => __( 'Enable bidirectional order fulfillment synchronization between WooCommerce and Square orders.', 'woocommerce-square' ),
```

Comment:
Not sure we need the phrase `order` here twice, maybe simplify this to:
```suggestion
						'description'       => __( 'Enable bidirectional fulfillment synchronization between WooCommerce and Square orders.', 'woocommerce-square' ),
```